docs/usage/dataset.md

```diff
@@ -126,3 +129,81 @@ Example of `class-descriptions.csv`
 
 ### DeLTA-Mark object detection
 You can download data of this format by using [Delta-Mark](https://delta.leapmind.io/mark/) service.
+
+
+### CamvidCustom
+
+It is data format based on [CamVid](http://mi.eng.cam.ac.uk/research/projects/VideoRec/CamVid/). We can see sample dataset from [SegNet-Turorial](https://github.com/alexgkendall/SegNet-Tutorial) repository. In CamvidCustom dataset format, training and annotation data are both binary image file.
+
+Place the following files and directory.
+
+- `label_colors.txt`: Class information file like under the example.
+- `train.txt`: The space separated pairs of data file path and annotation file path like under the example.
```

Comment:
Sorry, I didn't quite get the meaning of this description compare to the example given below.

Reply:
OK, I'm thinking better description...